apps/api/src/server/applications/application/representations/attachment/attachment.service.js

```diff
@@ -0,0 +1,9 @@
+import * as representationsRepository from '../../../../repositories/representation.repository.js';
+
+export const addAttachmentRepresentation = async (repId, documentId) => {
```

Comment:
Small naming question is this intended as in - add attachment to representation ?

Fine if it is.

Reply:
this follows the other naming conventions for the services.